### ISSUE TITLE
Downloader sources

### ISSUE THREAD

**Issue by benfishbus** [mrichards42/xword#125]
First, you could add the NYT "from our archives" puzzles to the downloader. They appear to be posted in puz format.

Second, release 0.7.0 is giving me 404 on The New Yorker, but I see source has been updated. Assuming another release is not imminent, can I pull/download default_sources.lua by itself?

**Comment by mrichards42** [comment]
Hi! RE: The New Yorker source, yes, you should be able to pull `default_sources.lua` and replace the file with that. You could also just change the url in the downloader preferences if you don't want to muck around with replacing the script file, although there's a related bug where the preferences don't save (but see #116 for a simple solution).

I'm happy to add more sources if you have a standard url that can be fetched based for a given date (i.e. there's year/month/day in some order in the url). It looks like the "from our archives" puzzles change randomly and don't follow a pattern though, so it might be tricky without writing a custom scraper which can be time-consuming.

**Comment by benfishbus** [comment]
Yes, the NYT archives puzzles change randomly. The links only load in a browser, too, so rudimentary scraping via curl/wget doesn't cut it. The whole DOM has to be loaded, then parsed for .puz links in that section. I'm sure this can be done, and served in an RSS feed for easy fetching, but it's beyond my ken (regardless how many times I poke around in it).
